xarray/core/dataarray.py

```diff
@@ -1369,6 +1369,83 @@ def real(self):
     @property
     def imag(self):
         return self._replace(self.variable.imag)
+    
```

Comment:
You still have the `tensordot` method defined -- you should delete it now that it's replaced by `dot` :)